tests/builder/axes_test.py

```diff
@@ -342,6 +342,145 @@ def test_mapping_using_axis_location_custom_parameter_on_instances(ufo_module):
     assert doc.axes[1].map == [(50, 224), (60, 384), (100, 448)]
 
 
+def test_mapping_using_axis_location_cp_on_masters_none(ufo_module):
+    # https://github.com/googlefonts/glyphsLib/issues/714
+    # https://github.com/googlefonts/glyphsLib/pull/810
+
+    # If custom parameters are disabled, they return None.
```

Comment:
I think the customParameters proxy object always returns None when a custom parameter that is looked up is not actually there (instead of throwing KeyError like a dict would normally do). So I'm not exactly sure what you're trying to do with your setting them explicitly to a `None` value below.

Reply:
The comment should maybe say "If custom parameters are missing, they return None."